examples/rcn/mnist_small.py

```diff
@@ -0,0 +1,310 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+import os
+import time
+
+import matplotlib.pyplot as plt
+
+# %%
+# %matplotlib inline
+import numpy as np
+from jax import numpy as jnp
+from jax import tree_util
+from load_data import get_mnist_data_iters
+from preproc import Preproc
+
+from pgmax.fg import graph, groups
+
+os.environ["XLA_PYTHON_CLIENT_ALLOCATOR"] = "platform"
+os.environ["XLA_PYTHON_CLIENT_PREALLOCATE"] = "false"
+
+# %% [markdown]
+# ## 1. Load the data
+#
+#
+# %%
+hps, vps = 12, 12
+train_size = 20
+test_size = 20
+data_dir = "/storage/users/skushagra/MNIST/"
+
+# +
+train_set, test_set = get_mnist_data_iters(data_dir, train_size, test_size)
+
+train_labels = -1 + np.zeros(len(train_set))
+for i in range(len(train_set)):
+    train_labels[i] = train_set[i][1]
+
+test_labels = -1 + np.zeros(len(test_set))
+for i in range(len(test_set)):
+    test_labels[i] = test_set[i][1]
+# -
+
+
+# %% [markdown]
+# ## 2. Load the model
+#
+#
+
+# %%
+directory = f"/storage/users/skushagra/pgmax_rcn_artifacts/model_science_{train_size}_{hps}_{vps}"
```

Comment:
Please include necessary data files inside `examples/example_data`, and structure the notebook in such a way that it can be run anywhere.